src/app/events/schemas/RecapSchema.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod'
+
+export const RecapSchema = z
+  .object({
+    'youtube-embed-url': z.string().url().optional(),
```

Comment:
Note to self as well, nothing to update here.

We should probably store only the YouTube video / playlist IDs in the Markdown files, and construct the URL in the front.

```ts
export const RecapSchema = z
  .object({
    'youtube-video-id': z.string().optional(),
    'youtube-playlist-id': z.string().optional(),
  })
  .strict()

```

Reply:
Thank you, Charly. I added it to the follow up dev work notes!